Plans_ideas.md

```diff
@@ -0,0 +1,56 @@
+# Plan of the Reproducible research code project
```

Comment:
This plan should probably not be in the main repo, so if you can delete this @ChendaHimalaya (but still of course have it somewhere, even in your repo is fine) that would be awesome.